LayoutTests/imported/w3c/web-platform-tests/trusted-types/HTMLScriptElement-in-xhtml-document.tentative.https-expected.txt

```diff
@@ -1,3 +1,5 @@
+CONSOLE MESSAGE: This requires a TrustedScript value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+
+Harness Error (TIMEOUT), message = null
```

Comment:
We don't want tests in a state where they have to time out. Can we avoid this?

Reply:
It's skipped in test expectations so it won't run. I'm working to make it so this test passes anyway but it requires a sizable changeset so doing it as followups.